gibbon-compiler/examples/layout_bench/DecisionTree.hs

```diff
@@ -0,0 +1,79 @@
+module Bench where
```

Comment:
Why is it `layout_bench` and `layout_benchmarks`? Should it be one directory?

Also, I propose adding more subdirectories inside `layout_benchmarks`, and putting most files inside them, so that the structure is clearer.